.eslintrc.json

```diff
@@ -48,6 +49,14 @@
       }
     ],
     "json/*": ["error", "allowComments"],
+    "@cspell/spellchecker": [
+      "warn",
```

Comment:
If there are 0 issues now, can't we change this to error?

Reply:
Done! I have made the change.